cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/client/AmazonEc2Client.java

```diff
@@ -211,4 +215,11 @@ public AuthorizeSecurityGroupIngressResult addIngress(AuthorizeSecurityGroupIngr
     public AuthorizeSecurityGroupEgressResult addEgress(AuthorizeSecurityGroupEgressRequest reguest) {
         return client.authorizeSecurityGroupEgress(reguest);
     }
+
+    // WIP
```

Comment:
Do we need this comment?

Reply:
Yes, we need it. The vpc creation will be finished in the next task, until this I don't want to delete this code snippet.